flask_admin/model/base.py

```diff
@@ -10,7 +10,7 @@
 from jinja2 import contextfunction
 from wtforms.fields import HiddenField
 from wtforms.fields.core import UnboundField
-from wtforms.validators import ValidationError, Required
+from wtforms.validators import ValidationError, InputRequired
```

Comment:
Good point: https://i.imgur.com/1vZTGbE.png

Also tested InputRequired and it worked for id=0. I'll merge it.